index.yaml

```diff
@@ -101,6 +101,13 @@ indexes:
   - name: last_updated
     direction: desc
 
+- kind: FeedbackMessageModel
```

Comment:
(So, the other moral here is, make sure before you submit the PR that you've manually exercised all possible code paths by playing with the devserver)